config/locales/it.yml

```diff
@@ -0,0 +1,741 @@
+      reason: "Ragione :"
+      reject: "Rifiutare la richiesta del cappello"
+      hator: "oppure"
+    approve_request:
+      approvedhatrequest: "La richiesta del cappello è stata approvata."
+  home:
+    index:
+      homerecentsdesc: "<em>Gli <a href=\"/newest\">ultimi</a> articoli con una frazione casuale di articoli che non hanno ancora raggiunto la pagina principale.</em>"
+  invitations:
+    build:
+      buildinvtitle: "Richiedi un invito"
+      buildinvtext: "<p>Se non conosci o non sai come trovare un <a href=\"/u/\">utente iscritto</a> a cui richiedere l'invito, puoi richiedere un invito pubblicamente. Riceverai direttamente l'invito se ti sarà inviato.</p><p>Il tuo indirizzo e-mail deve essere valido e verificato visitando l'URL che ti sarà inviato prima che la tua domanda sia elaborata. Il tuo indirizzo e-mail non sarà visibile da nessuno ad eccezione dei moderatori. </p>"
+      buildinvname: "Nome :"
+      buildinvemail: "Indirizzo e-mail :"
+      buildinvurl: "URL :"
+      buildinvverificationurl: "URL verificabile (sito web personale, GitHub, etc.)"
+      buildinvrequest: "Richiedi l'invito"
+    index:
+      requestedinvtitle: "Richieste di invito"
+      requestedinvsummary: "Queste persone hanno richiesto un invito ed hanno confermato il loro indirizzo e-mail. Se riconosci qualcuno, inviagli un invito e accetta la domanda. La spam sarà eliminata da un moderatore."
+      requestedinvdate: "Data/Ora"
+      requestedinvname: "Nome"
+      requestedinvmemo: "Nota"
+      sendinvitation: "Invia l'invito"
+      confirminvitation: "Sei sicuro di voler invitare l'utente e accettare la richiesta?"
+      deleteinvitation: "Cancella"
+      confirmdelete: "Sei sicuro di voler rifiutare la domanda?"
+      noinvitation: "Nessun invito"
+  invitation_mailer:
+    invitation:
+      title: "Buongiorno %{email},"
+      invitedto: "L'utente %{user} ti ha invitato a %{appname}"
+      createaccount: "Per accettare un invito e creare un account, clicca sul seguente link :"
+  invitation_request_mailer:
+    invitation_request:
+      title: "Buongiorno %{email},"
+      invitedto: "L'utente con l'IP %{userip} ha richiesto un invito a %{appname}."
+      name: "Nome : %{name}"
+      email: "E-mail : %{email} (non sarà visualizzata da altri utenti)"
+      memo: "Informazioni : %{memo}"
+      warning: "Se sei tu, clicca l'URL sottostante per confermare la tua richiesta di iscrizione et l'invio ad altri utenti."
+      deletemessage: "Se non sei tu, puoi semplicemente ignorare questo messaggio."
+  login:
+    index:
+      login: "Login"
+      email: "E-mail o nome utente :"
+      password: "Password :"
+      forgotdeleted: "Password dimenticata? "
+      resetpassword: "Cambia password"
+      notuseryet: "Non sei ancora iscritto? L'iscrizione è solo tramite invito per evitare lo spam e migliorare la tracciabilità. Se conosci <a href=\"/u/\">%{acurrentuser}</a> del sito, richiedigli un invito"
```

Comment:
L'iscrizione è possibile solo tramite

tracciabilità fa un po' paura. Riformulate menzionando qualcosa di moderazione magari

Reply:
Spazio prima dei due punti in 'Password'

Reply:
> L'iscrizione è possibile solo tramite
> 
> tracciabilità fa un po' paura. Riformulate menzionando qualcosa di moderazione magari

Sostituire tracciabilità con moderabilità?

Reply:
moderabilità non è una parola. Moderazione va benissimo

Reply:
Stando sempre a Treccani, [lo è](http://www.treccani.it/vocabolario/moderabilita/). Moderazione è adeguato, ma ha un campo semantico meno specifico.
@riggraz che dici?

Reply:
Moderabilità è corretto però è molto poco usato, neanch'io lo conoscevo. Per questo motivo userei moderazione. Inoltre cambierei 'migliorare' con 'facilitare':
'L'iscrizione è solo tramite invito per evitare lo spam e facilitare la moderazione'